packages/flame/lib/src/components/component.dart

```diff
@@ -122,10 +122,14 @@ class Component with Loadable {
   /// your state considering this.
   /// All components in the tree are always updated by the same amount. The time
   /// each one takes to update adds up to the next update cycle.
-  @mustCallSuper
-  void update(double dt) {
+  void update(double dt) {}
+
+  void updateTree(double dt, {bool callUpdate = true}) {
```

Comment:
We should definitely have a doc-comment here, because the purpose of this method (and how it is different from `update()`) will not be immediately clear to a reader.